src/napari_blob_detection/_widget.py

```diff
@@ -15,30 +15,53 @@ class Dimensionality(Enum):
 
 class Algorithm(Enum):
     DOG = auto()
-    DOH = auto()
```

Comment:
Did you remove this because it only detects 2D features?

Reply:
yes